### ISSUE TITLE
Error 1025

### ISSUE THREAD

**Issue by OkanSen** [ShikyoKira/Project-New-Reign---Nemesis-Main#247]
Hi, I've recently tried switching from FNIS to Nemesis and I'm having the some problems. Upon downloading and setting it up the first time it worked just fine. It generated 6400ish animations. Then I rerun it again to make sure of some file changes, then it's constantly giving this error. And it is not the duplicate of #195, output is as follows;

ERROR(6002): An exception has occured while patching. Report to Nemesis' author immediately
File: None
Message: boost::filesystem::remove: The process cannot access the file because it is being used by another process: "temp_behaviors\xml_1stperson\magicbehavior.xml"

Failed to generate behavior

Even if it is the same reason for #195, I've done what you have stated in that thread "re run the operation", but it always gives the same error.

If it'll help in any way, I removed FNIS PCEA2, Fores New Idle animations, and Sexy Move since Nemesis is not compatible with them.

----------------------------------------------------

NEW UPDATE!!!
I have restarted my PC and tried again, now the process has started but it fails due to a new error. here's the output:

ERROR(1025): Failed to output xml file
File: d:\program files (x86)\steam\steamapps\common\skyrim special edition\data\meshes\animationsetdatasinglefile

And I'm not sure what to do for this.

UPDATE: I've checked some threads with the same error code but they had different issues. I couldn't find any other errors with my issue. I've tried running both MO2 and Nemesis as Admin but it doesn't change the output.

**Comment by ShikyoKira** [comment]
Duplicate of #173 
different error code but same underlying problem. Until now I couldn't duplicate this issue here. How about you try install Nemesis manually in Data directory and run it through MO2?

**Comment by OkanSen** [comment]
Oh I'm so sorry for not writing here. So here's how I solved the issue and a possible issue as well.

Whenever I tried to generate animations, Nemesis gave an error saying some file was being used by another process. So, I checked what it was, and it turned out to be hkcmxd.exe or what it's correct name was. Trying to kill it from Task manager wasn't possible since it was a hidden app. So I restarted the PC. 

Now I could only run Nemesis only once, while running MO2 and Nemesis as admin, per restart session. It generated successfully. But let's say if I removed an animation mod, or added a new one and then started generation process again, it gave the same error again. Because hkcmxd.exe was still running. I have to restart/sign out for every animation generation. 

I'm not going to add or remove any more animation related mods in the near future, but I wanted to highlight the little issue.

Thanks for the mod as well, it's really much better than TK Dodge since it reacts quickly to the dodge input button, and adding a lot to combat. Appreciate the hard work!

**Comment by ShikyoKira** [comment]
Oh my, this is an awesome finding bro. I can't thank you enough for this. This might just be the clue I need to solve this. It has been quite some time trying to deal with this for me but after all these time, the result is mixed, some fine, some don't. I fall in the fine group, so I have never been able to recreate this issue. It have something to do with the task killing part that isn't working right

Do you mind if we bring this discussion over to discord? Its easier for me there. Drop me your discord username, I'll pm you the invitation

I'll close this issue btw for it is a duplicate

**Comment by OkanSen** [comment]
Of course, I'll be glad to help in any way!

Haytham#8585

"It have something to do with the task killing part that isn't working right" 

Yes, it was about that, there are several other ways to find and kill a hidden app, but I don't want to dive that deep. So that's why I just sign out or restart. Feel free to PM me any time I'll reply back whenever possible ^^
